components/prism-cil.js

```diff
@@ -0,0 +1,27 @@
+Prism.languages.cil = {
+	'comment': /\/\/.*/,
+	
+	'string': {
+		pattern: /(?:["'])(?:\\(?:\r\n|[\s\S])|(?!\1)[^\\\r\n])*\1/,
```

Comment:
The first group has to be capturing otherwise, strings won't be highlighted.
Maybe a string test would be good?